app/src/main/java/com/github/se/polyfit/viewmodel/meal/MealViewModel.kt

```diff
@@ -10,7 +10,7 @@ import com.github.se.polyfit.model.meal.Meal
 class MealViewModel(
     private val userId: String,
     firebaseID: String = "",
-    initialMeal: Meal? = null,
+    var initialMeal: Meal? = null,
```

Comment:
Can you explain why we are using it as a var?

Reply:
I was waiting for you to commit your changes and did not want to break any implementation before that.